File: DotNetGrc/Grc/Ast/Node/Expr/ExprIntegerT.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Grc.Ast.Visitor;

namespace Grc.Ast.Node.Expr
{
	public partial class ExprIntegerT : ExprBase
	{
		private readonly string integer;

		private readonly int line;
		private readonly int pos;

		public string Integer { get { return integer; } }

		public override int Line { get { return line; } }

		public override int Pos { get { return pos; } }

		public ExprIntegerT(string integer, int line, int pos)
		{
			this.integer = integer;

			this.line = line;
			this.pos = pos;
		}

		public override void Accept(IVisitor v)
		{
			v.Visit(this);
		}

		public override string Text
		{
			get { return integer; }
		}
	}
}

File: Parser/src/main/java/k31/grc/ast/node/expr/ExprIntegerT.java
package k31.grc.ast.node.expr;

import k31.grc.ast.visitor.Visitor;

public class ExprIntegerT extends ExprBase {

	public ExprIntegerT(String text) {
		super(text);
	}

	@Override
	public void accept(Visitor v) {

		v.visit(this);
	}
}
